ocaml/idl/datamodel_common.ml

```diff
@@ -8,7 +8,7 @@ open Datamodel_roles
               When introducing a new release, bump the schema minor version to the next hundred
               to leave a gap for potential hotfixes needing to increment the schema version.*)
 let schema_major_vsn = 5
-let schema_minor_vsn = 501
+let schema_minor_vsn = 502
```

Comment:
@psafont said not to bother with bumping these versions? ( however the 'add a new record' guide says to )

Reply:
I mispoke, the comment above the field is pretty clear :)